distribution/lib/Standard/Google/0.0.0-dev/src/Google_Sheets_Workbook.enso

```diff
@@ -167,14 +169,14 @@ type Google_API_Error
     to_display_text self =
         self.details.at 'message'
 
-private _handle_exceptions ~action = Empty_Sheet.handle_java_exception <|
+private _handle_exceptions ~action =
```

Comment:
We could inline the Empty_Sheet handler here and then the exception will be from correct JVM.